### ISSUE TITLE
[9.0] KeyError: 'mar.'

### ISSUE THREAD

**Issue by max3903** [openeducat/openeducat_erp#60]
On version 9.0, when I try to generate the timetables, I get the following error:

```
Traceback (most recent call last):
  File "/home/max/odoo/9.0/git/odoo/openerp/http.py", line 605, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/home/max/odoo/9.0/git/odoo/openerp/http.py", line 642, in dispatch
    result = self._call_function(**self.params)
  File "/home/max/odoo/9.0/git/odoo/openerp/http.py", line 316, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/home/max/odoo/9.0/git/odoo/openerp/service/model.py", line 118, in wrapper
    return f(dbname, *args, **kwargs)
  File "/home/max/odoo/9.0/git/odoo/openerp/http.py", line 309, in checked_call
    result = self.endpoint(*a, **kw)
  File "/home/max/odoo/9.0/git/odoo/openerp/http.py", line 893, in __call__
    return self.method(*args, **kw)
  File "/home/max/odoo/9.0/git/odoo/openerp/http.py", line 471, in response_wrap
    response = f(*args, **kw)
  File "/home/max/odoo/9.0/git/odoo/addons/web/controllers/main.py", line 896, in call_button
    action = self._call_kw(model, method, args, {})
  File "/home/max/odoo/9.0/git/odoo/addons/web/controllers/main.py", line 884, in _call_kw
    return getattr(request.registry.get(model), method)(request.cr, request.uid, *args, **kwargs)
  File "/home/max/odoo/9.0/git/odoo/openerp/api.py", line 238, in wrapper
    return old_api(self, *args, **kwargs)
  File "/home/max/odoo/9.0/git/odoo/openerp/api.py", line 409, in old_api
    result = new_api(recs, *args, **kwargs)
  File "/home/max/odoo/9.0/git/odoo/openerp/api.py", line 413, in new_api
    result = [method(rec, *args, **kwargs) for rec in self]
  File "/home/max/odoo/9.0/git/openeducat_erp/openeducat_timetable/wizard/generate_timetable.py", line 121, in act_gen_time_table
    st_day = week_number[st_date.strftime('%a')]
KeyError: 'mar.'
```

**Comment by ajaychauhan727** [comment]
I can't get this traceback on 9.0 branch. Can you tell revision no. & steps to reproduce it.

**Comment by max3903** [comment]
@ajaychauhan727 I am using commit 28b9c1348a86331dae780d13b69143e217da26d8 and I guess this is a localization issue. "mar" refers to tuesday in french.

**Comment by parthivgls** [comment]
Fixed by #94
